easybuild/easyconfigs/p/pdbfixer/pdbfixer-1.7-foss-2020b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'pdbfixer'
+version = '1.7'
+
+homepage = 'https://github.com/openmm/pdbfixer'
+description = """PDBFixer is an easy to use application for fixing problems 
+in Protein Data Bank files in preparation for simulating them."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/openmm/pdbfixer/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['a0bef3c52a7bbe69a6aea5333f51f3e7d158339be5829aed19b0344bd66d4eea']  # v1.7.tar.gz
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('OpenMM', '7.5.1'),
+]
+
+use_pip = True
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s/'],
+}
+
+# sanity_check_commands = ['%(namelower)s-count --help']
+
+# sanity_pip_check = True
+
+# options = {'modulename': '%(name)s'}
```

Comment:
Remove

Reply:
Done. That was from a template file I used.